skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/PenService.kt

```diff
@@ -143,10 +143,12 @@ class PenService(config: Config, authService: AzureADService) : ServiceStatus {
     suspend fun sendbrev(
         call: ApplicationCall,
         sendRedigerbartBrevRequest: SendRedigerbartBrevRequest,
+        distribuer: Boolean,
     ): ServiceResult<Pen.BestillBrevResponse> =
         client.post(call, "brev/skribenten/sendbrev") {
             setBody(sendRedigerbartBrevRequest)
             contentType(ContentType.Application.Json)
+            url { parameters.append("distribuer", distribuer.toString()) }
```

Comment:
Hvorfor ikke gjøre distribuer til en del av SendRedigerbartBrevRequest? evt også rename SendRedigerbartBrevRequest

Reply:
Jeg tror jeg er enig med @hconhegg her - hvis vi kan gjøre det likt som lås for redigering.